rocketpy/mathutils/function.py

```diff
@@ -1144,6 +1144,49 @@ def low_pass_filter(self, alpha, file_path=None):
             title=self.title,
         )
 
+    def remove_outliers_iqr(self, threshold=1.5):
+        """Remove outliers from the Function source using the interquartile
```

Comment:
It should be documented, either here or on a "notes" section (compare `set_discrete_based_on_model`) that the method only works if the `Function` is array based.

Reply:
Good point. Solved here: [5987bd8](https://github.com/RocketPy-Team/RocketPy/pull/554/commits/5987bd8fb267daf95a6ed345849c3b218dcfb838)